src/main/java/com/mk/cti/controller/PhoneController.java

```diff
@@ -35,14 +35,14 @@ public PhoneController(PhoneUtility phoneUtility, PhoneResponseService phoneResp
 
         PhoneResponse phoneResponse = phoneResponseService.findByPhoneNumber(phoneNumber);
 
-        if(phoneResponse == null)
-        {
+        if(phoneResponse == null) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)